### ISSUE TITLE
service providers: add sector

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#225]
We should add a field for sector to both service providers and contractors (with perhaps a slightly different label?)

How about this list?
```
Community
E-Commerce
Education
Enterprise
Entertainment
Financial
Government
Healthcare
Legal
Media
Non-profit
Publishing
Religious
Small - Medium business
Startups
Technology
```

**Comment by goldmantx** [comment]
@jenlampton Looks like a great start to me. I assume we want to keep this list on the smaller side to start? I compared against Drupl.org list and there is a bunch more but might seem like overkill. We could always add as needed...

**Comment by jenlampton** [comment]
Yeah, I started from the d.o list and trimmed off a few of the less-popular selections. We can add them back as/if requested.

**Comment by goldmantx** [comment]
Agreed. Looks good!

**Comment by docwilmot** [comment]
Is this discriminating against the people who provide those less popular services?

**Comment by jenlampton** [comment]
It's not discriminating as they can still add a company node and select any other things they might do. There are lots of nice big categories like "small - medium business" that are good catch-alls so that everyone will be able to choose at least one.

**Comment by klonos** [comment]
...we could add a "my sector is not listed" link that will send an email to us so we can give people the chance to tell us if we've left an important sector out.

**Comment by jenlampton** [comment]
yep, good idea @klonos ! 👍
